indico/modules/events/surveys/templates/manage_survey.html

```diff
@@ -38,7 +38,7 @@
                 {% trans %}Submissions{% endtrans %}
             </div>
             <div class="i-box-buttons">
-                <a href="#" class="i-button">
+                <a href="{{ url_for('.survey_results', survey) }}" class="i-button">
                     {% trans %}Show results{% endtrans %}
```

Comment:
Could you add some icon to this button? The reason why is that there are two additional buttons for exporting and deleting submissions and both of them have icons.